app/controllers/check_records/search_controller.rb

```diff
@@ -7,6 +7,11 @@ def new
     end
 
     def show
+      unless params["search"].key?("date_of_birth(1i)") &&
```

Comment:
Presumably it's possible to hit this route without a `search` key set in the params? In that case, this code will raise an error.